src/main/java/tds/exam/results/mappers/TestMapper.java

```diff
@@ -38,4 +51,53 @@ private static String parseBankKeyFromId(final String itemId) {
 
         return itemId.split("-")[0];
     }
+
+    private static long parseLatestAcademicYear(final String academicYearString) {
+        // If this is a range date, pick the latest year
+        if (academicYearString.contains("-")) {
+            return Long.parseLong(academicYearString.split("-")[1]);
+        }
+
+        return Long.parseLong(academicYearString);
+    }
+
+    private static String createGradeStringFromGrades(final List<String> grades) {
+        if (grades.size() == 0) {
+            return StringUtils.EMPTY;
+        } else if (grades.size() == 1) {
+            return grades.get(0);
+        }
+
+        int minGrade = Integer.MAX_VALUE;
+        int maxGrade = Integer.MIN_VALUE;
+        int numOfIntegerGrades = 0;
```

Comment:
Can we remove this int and simply use `intGrades.size()` instead?  Since they will always be the same value.  Or if you would prefer to keep the int can we move it right above line 88 (`int numOfIntegerGrades = intGrades.size()`)?